.rubocop.yml

```diff
@@ -661,9 +606,9 @@ Lint/EmptyConditionalBody:
   Enabled: false
 
 # Disabling for now
-Lint/EmptyEnsure:
-  Enabled: false
-  # AutoCorrect: true
+# Lint/EmptyEnsure:
```

Comment:
Do we want to remove this entirely vs comment out?

Reply:
Great catch! Fixed in: 03134c1433aa21e2ffef9cc67cc47ef1ce3f92f4